content/en/synthetics/browser_tests/_index.md

```diff
@@ -110,19 +134,23 @@ A notification is sent according to the set of alerting conditions. To configure
 3. Specify a renotification frequency. To prevent renotification on failing tests, leave the option as `Never renotify if the monitor has not been resolved`.
 4. Click **Save Details and Record Test**.
 
+### Set permissions
```

Comment:
can you please move the section ##permissions at the end of the file to this place?

Reply:
But this is the last step in the test creation workflow and we link to the ## Permissions below !

Reply:
why not moving the section which is at the bottom to this place?

Reply:
imo one is calling out an action for users to set their permissions on the test vs. the broader "Permissions" description that we have in the paragraph about permissions. I'd also leave this step in to be consistent with Step 8 in the UI.